src/main/java/doubly_linked_list/Controller.java

```diff
@@ -1,2 +1,130 @@
-package doubly_linked_list;public class Controller {
+package doubly_linked_list;
+
+import java.util.Scanner;
+
+public class Controller {
+    private Scanner scanner;
+    private DoublyListService service;
+
+    public Controller() {
+        scanner = new Scanner(System.in);
+        service = new DoublyListService();
+    }
+
+    public void userInterface() {
+        int choose;
+        System.out.println("Enter a sequence of numbers: (input ending - number 0)");
+        Node head = service.createList();
+        while (true) {
+            System.out.println("What do you want to do:\n1-Create new list\n2-Delete item;\n3-Add element BEFORE;" +
+                    "\n4-Add element AFTER;" + "\n5-Show all elements of a sequence;\n6-Length of the list;\n" +
+                    "7-Is the list symmetrical?\n8-List to array\n0-Exit;");
+            choose = scanner.nextInt();
+            switch (choose) {
+                case 1 -> {
+                    createNewList(head);
+                }
+                case 2 -> {
+                    removeElement(head);
+                }
+                case 3 -> {
+                    before(head);
+                }
+                case 4 -> {
+                    after(head);
+                }
+                case 5 -> print(head);
+                case 6 -> length(head);
+                case 7 -> isSymmetric(head);
+                case 8 -> toArray(head);
+                case 0 -> System.exit(1);
+            }
+        }
+    }
+
+    private void print(Node head) {
+        service.printList(head);
+    }
+
+
+    private void removeElement(Node head) { //should be remade
```

Comment:
I think you should return head of list in a list-modifying methods such as ```create```, ```remove```, ```insertBefore``` and ```insertAfter```. You can use syntax like: ```case 2 -> head = removeElement(head);```
Also it will be useful to add service layer method like ```clear(Node top)```. This method will be needed to remove all of the elements in current list. And you should call this method when you want to create new list in the loop.